src/json-patch-ot-agent.js

```diff
@@ -72,3 +73,14 @@ JSONPatchOTAgent.applyOT = function(callback){
         callback(obj, consecutivePatch);
 	};
 };
+
+/**
+ * Reset queue internals and object to new, given state
+ * @param obj object to apply new state to
+ * @param newState versioned object representing desired state along with versions
+ */
+JSONPatchOTAgent.prototype.reset = function(obj, newState){
```

Comment:
It is reset  at
https://github.com/PuppetJs/JSON-Patch-OT-agent/blob/gh-pages/src/json-patch-ot-agent.js#L59 and  athttps://github.com/PuppetJs/JSON-Patch-OT-agent/blob/gh-pages/src/json-patch-ot-agent.js#L62